lib/build-cloud.rb

```diff
@@ -220,16 +223,31 @@ def recursive_interpolate_config(h)
         when String
 
             while h =~ /%\{(.+?)\}/
```

Comment:
Probably need to include `h` in there somewhere too. So the regex actually operates on something ;)